docs/usage/configuration-options.md

```diff
@@ -1679,6 +1684,30 @@ Use this field to restrict rules to a particular language. e.g.
 }
 ```
 
+### matchCategories
```

Comment:
Move the new section for `matchCategories` upwards, so it's sorted A through Z:

- `matchCategories` -> first different letter is an C
- `matchLanguages` -> first different letter is an L